dist/less/ds6/variables.less

```diff
@@ -1,9 +1,62 @@
 @import '../variables-common.less';
 
-@ds6-color-core-default: #111820;
-@ds6-color-core-light: #767676;
-@ds6-color-core-link: #006efc;
-@ds6-color-core-visited: #6666d1;
+// Colours
+//----------------------
 
-@ds6-color-interface-button-blue: @ds6-color-core-link;
-@ds6-color-interface-button-grey: #e5e5e5;
+@ds6-color-default: #111820;
+@ds6-color-light: #767676;
+
+@ds6-color-link: #006efc;
+@ds6-color-visited: #6666d1;
+
+@ds6-color-disabled: #e5e5e5;
+
+@ds6-color-critical: #c9002c;
+@ds6-color-success: #147133;
+
+@ds6-color-button-blue: @ds6-color-link;
+@ds6-color-button-grey: @ds6-color-disabled;
```

Comment:
In my brain these would be reversed, where the description of the color precedes the "type" of color it is:

```
@ds6-color-button-blue: #006efc;
@ds6-color-button-grey: #e5e5e5;
...
@ds6-color-link: @ds6-color-button-blue;
@ds6-color-disabled: @ds6-color-button-grey;
```
Thoughts?

Reply:
Ah! Is it because one is button v. link?

Reply:
I think I just have them this way around because color-link and color-disabled are core colors. I'm tempted to change their names to color-core-link and color-core-disabled to make that more obvious.